servicetalk-transport-netty-internal/src/main/java/io/servicetalk/transport/netty/internal/WriteStreamSubscriber.java

```diff
@@ -337,7 +337,8 @@ public final ChannelPromise removeListeners(
 
         boolean isWritable() {
             assert channel.eventLoop().inEventLoop();
-            return !hasAnyFlags(CHANNEL_CLOSED, SUBSCRIBER_TERMINATED, SOURCE_TERMINATED);
+            return !hasAnyFlags(CHANNEL_CLOSED, SUBSCRIBER_TERMINATED, SOURCE_TERMINATED,
```

Comment:
nit: this could be `state != 0` based upon the current state machine. (understood if you want to be more explicit about this with the current code).